### PR TITLE
Add support for bearer token scala-akka #5067

### PR DESCRIPTION
**Pull request by jaroslawZawila**
Add support for bearer token authorization for scala-akka client. #5067

### CODE REVIEW

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

File: modules/openapi-generator/src/main/resources/scala-akka-client/api.mustache [hunk 1]
```diff
@@ -2,52 +2,51 @@
 package {{package}}
 
 {{#imports}}
-import {{import}}
+    import {{import}}
```
Comment (wing328) [inline modules/openapi-generator/src/main/resources/scala-akka-client/api.mustache]: @jaroslawZawila Did you add the spaces intentionally? what about line 7 to 9 ?

Comment (jaroslawZawila) [reply modules/openapi-generator/src/main/resources/scala-akka-client/api.mustache]: I have removed the spaces.

**Comment by jaroslawZawila** [comment]
I think it should be good now.

File: modules/openapi-generator/src/main/resources/scala-akka-client/api.mustache [hunk 2]
```diff
@@ -9,45 +9,44 @@ import {{mainPackage}}.core.CollectionFormats._
...
-  {{/responseHeaders.isEmpty}}
-
-{{/operation}}
-
-  {{#unknownStatusCodes}}
-  ApiInvoker.addCustomStatusCode({{{value}}}, isSuccess = false)
-  {{/unknownStatusCodes}}
-
-}
+    object {{classname}} {
```
Comment (wing328) [inline modules/openapi-generator/src/main/resources/scala-akka-client/api.mustache]: @jaroslawZawila  Looks like this line is not indented correctly.

**Comment by wing328** [comment]
Can we keep the indention as 2-space instead of 4-space?

Code formatting can be done in the file post-processing hook or after code generation using various plug-ins or code formatting tool.

**Comment by jimschubert** [comment]
I've resolved the issues mentioned above with indentation. I've assigned to myself, and I think this can be merged once CI is successful.

**Comment by wing328** [comment]
@jaroslawZawila thanks for the PR, which has been included in the 4.3.0 release: https://twitter.com/oas_generator/status/1243455743937789952

**Pull request merged**